app/views/works/_work_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render partial: 'work_form',
+locals: {
+  model: @work
+}
+%>
+
+<%= form_with model: @work, class: 'work-form' do |f| %>
+
+<h3><%= form_title %></h3>
+
+<div class="work-form">
+  <%= f.label :title %>
+  <%= f.text_field :title %>
+  <%= f.label :creator %>
+  <%= f.text_field :creator %>
+  <%= f.label :publication_year %>
+  <%= f.text_field :publication_year %>
+  <%= f.label :description %>
+  <%= f.text_field :description %>
+  <%= f.label :category %>
+  <%= f.select :category, Work::CATEGORIES %>
+  <%= f.submit action_type, class: "work btn" %>
+  <% end %>
+</div>
```

Comment:
You have an `end` on line 23 that ends the `form_with` block that started on line seven... And you have a div that starts on line 11 and ends on line 24. These aren't neatly nested with each other -- you should be mindful that your end should be outside of anything that it includes fully, otherwise it isn't proper syntax and you'll get negative side-effects